package.json

```diff
@@ -84,5 +84,9 @@
     "stylelint-config-recommended": "^6.0.0",
     "stylelint-config-sass-guidelines": "^9.0.1",
     "stylelint-no-unsupported-browser-features": "^5.0.2"
+  },
+  "dependencies": {
+    "react-slick": "^0.28.1",
```

Comment:
Yes.  One is the JS, the other CSS

Reply:
wait, I'm confused. Maybe I'm missing something around this. So, we've effectively "forked" the scss by copying and pasting the scss in? If the intention is to "freeze" the version, we should reflect that in the `package.json`
```suggestion
    "react-slick": "0.28.1",
```

> If we are against linking to the foundational Sass, then we can copy that in, but we then should freeze the version of the react-slick component we are currently using.

Reply:
has this been addressed @visionbegin @ecarlisle ? I'd recommend freezing the version via specifying the package number explicitly

Reply:
If we lock slick, we should also lock slick-carousel.  If not, they should at least have the same upgrade markers (~, ^, etc.) in similar locations.

Reply:
@JackHowa We agreed yesterday that we wouldn't freese `react-slick`.  Instead, we would import the base scss file from `slick-carousel` but copy in the theme file from that package so that we can integrate it into the advanced styling.